fplll/gso.cpp

```diff
@@ -71,7 +71,7 @@ template <class ZT, class FT> void MatGSO<ZT, FT>::discover_row()
   {
     for (int j = 0; j <= i; j++)
     {
-      b[i].dot_product(g(i, j), b[j], n_known_cols);
+      b[i].dot_product(g(i, j), b[j], 0, n_known_cols);
```

Comment:
Was this a bug in the gso implementation?

Reply:
No, as you have remarked in a next conversation, I have modified the way to call `dot_product`. I will discuss in this next conversation what I can propose to change this status.